patches/net/minecraft/world/level/block/state/BlockBehaviour.java.patch

```diff
@@ -34,13 +34,13 @@
          return this.isRandomlyTicking;
      }
  
-+    @Deprecated //Forge: Use more sensitive version {@link IForgeBlockState#getSoundType(IWorldReader, BlockPos, Entity) }
++    @Deprecated //Neo: Use more sensitive version {@link net.neoforged.neoforge.common.extensions.IBlockStateExtension#getSoundType(LevelReader, BlockPos, Entity) }
      protected SoundType getSoundType(BlockState p_320941_) {
          return this.soundType;
      }
  
 +    @Deprecated
-+    /** @deprecated NeoForge: use {@link net.neoforged.neoforge.common.extensions.IBlockExtension#getCloneItemStack(BlockState, LevelReader, BlockPos, Player, boolean) player-sensitive version} */
++    /** @deprecated Neo: use {@link net.neoforged.neoforge.common.extensions.IBlockExtension#getCloneItemStack(LevelReader level, BlockPos pos, BlockState state, boolean includeData, Player player) player-sensitive version} */
```

Comment:
```suggestion
+    /** @deprecated Neo: use {@link net.neoforged.neoforge.common.extensions.IBlockExtension#getCloneItemStack(LevelReader, BlockPos, BlockState, boolean, Player) player-sensitive version} */
```

Sorry, I know I'm the one who provided the suggestion, but I forgot to remove the parameter names.